extensions/schema-compare/package.json

```diff
@@ -34,6 +34,20 @@
 				}
 			}
 		],
+		"languages": [
+			{
+				"id": "scmp",
+				"filenames": [
+					"Schema Compare"
```

Comment:
localize the strings in package.json

Reply:
Localized everything other than file name. Thanks for the catch.